pkg/config/configurable/set.go

```diff
@@ -0,0 +1,60 @@
+// SPDX-FileCopyrightText: 2020-present Open Networking Foundation <info@opennetworking.org>
+//
+// SPDX-License-Identifier: LicenseRef-ONF-Member-1.0
+
+package configurable
+
+import (
+	"github.com/onosproject/onos-ric-sdk-go/pkg/config/utils"
+	pb "github.com/openconfig/gnmi/proto/gnmi"
+)
+
+func (c *Config) Set(req SetRequest) (SetResponse, error) {
```

Comment:
This function will be called by the gNMI Agent for Set requests

Reply:
Yes, I got this and the get stuff in configurable... what threw me off track was that the RegisterConfigurable looked to be some internal mechanism as it exposes too much, IMO.... and does not appear to be launching the agent, lazily. The agent should not start unless `Register(Configurable)` is invoked.